manuscript/manuscript.Rmd

```diff
@@ -441,21 +441,14 @@ However, the limitations of classical methods such as optimal control frequently
 Complexity comes not only from the ecological processes but also the available actions.
 Deep RL agents have proven remarkably effective in handling such complexity, particularly when leveraging immense computing resources increasingly available through advances in hardware and software [@gpu_computing]. 
 
+<!-- RL for behavioral ecology is not the same as RL for conservation -->
 This paper does not set the precedent as the first application of RL to ecology.
-In particular, there have been a number of studies applying RL to behavioral ecology, typically with multi-agent environments [@wang_reinforcement_2020, @frankenhuis_enriching_2019, @perolat_multi-agent_2017].
+There have been a number of studies applying RL to behavioral ecology, typically with multi-agent environments [@wang_reinforcement_2020, @frankenhuis_enriching_2019, @perolat_multi-agent_2017].
 Yet, it is important to distinguish the aim of these behavioral studies from the aim of applying RL to conservation management.
-In behavioral ecology RL studies, the authors employ RL agents to represent organisms.
-Assuming that animals try to behave optimally under some objective, these authors then draw conclusions about animal behavior and evolution by observing how the RL agents learn.
-When applying deep RL to conservation management, we do not make a similar logical leap to assuming that animals behave like RL agents.
-We instead propose using RL as a way to search for solutions to unsolved decision-making problems in conservation. 
-
-<!-- Model free is great bc we don't know the true models -->
-The rapidly expanding class of model-free RL algorithms is particularly appealing given the ubiquitous presence of model uncertainty in ecological dynamics.
-Rarely do we know the underlying functional forms for ecological processes.
-Methods which must first assume something about the structure or functional form of a process before estimating the corresponding parameter can only ever be as good as those structural assumptions.
-Frequently available ecological data is insufficient to distinguish between possible alternative models [@knape_are_2012], or the correct model may be non-identifiable with any amount of data.
-Model-free RL approaches offer a powerful solution for this thorny issue.
-Model-free algorithms have proven successful at learning effective policies even when the underlying model is difficult or impossible to learn [@Pong2020], as long as simulations of possible mechanisms are available.
+In behavioral ecology studies, RL agents are used to represent organisms.
+Assuming that animals attempt to maximize some cumulative reward like RL agents, the authors of these studies draw conclusions about animal behavior and evolution by observing how the RL agents perform.
```

Comment:
this is good, but I think we can expand a little more on it.  i.e. as phrased this doesn't sound like a terrible assumption -- i.e. that animals attempt to maximize a cumulative reward.  This is really a distinctly multi-agent problem.

Some behavioral ecology isn't multi-agent -- i.e. optimal forging theory (or optimal life-history theory), you assume evolution has tuned behavior to maximize fitness.  In this case it doesn't matter if you use classical optimization or RL to find the optimum, because your finding a global optimum.  

It's the multi-agent case that's the problem, where each agent is trying to optimize their score against the other agent.  This case is problematic because suddenly there's (usually) no definition of what behavior is 'optimal' globally -- your optimal strategy is contingent on what your opponent is doing.   These papers don't just assume animals are maximizing something, (actually they will usually concede they are not), but that the animals or evolution is 'learning' in the same way the RL learns.  i.e. you aren't just assuming the 'end result' is the same (since you never reach an end result), but that RL is actually a model of the dynamic process of animal learning / updates, or a model of behavioral evolution, which is a much much stronger assumption than to our use of saying "it's just a black box to find an optimum, we make no assertion that the way the black box finds that optimum is in any way analogous to how animals learn or how evolution works".

Reply:
okay I'll try a shorter stab at this soon.

Reply:
Yeah I get your points here